adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -64,19 +69,17 @@ + (void)setUpWithConfiguration:(nonnull GADMediationServerConfiguration *)config
         appIds);
     GADMPangleLog(@"Configuring Pangle SDK with the app ID:%@", appID);
   }
-
-  BUAdSDKConfiguration *sdkConfiguration = [BUAdSDKConfiguration configuration];
-  sdkConfiguration.territory = BUAdSDKTerritory_NO_CN;
-  sdkConfiguration.appID = appID;
-  sdkConfiguration.GDPR = @(_gdpr);
-  sdkConfiguration.CCPA = @(_ccpa);
-  [BUAdSDKManager startWithAsyncCompletionHandler:^(BOOL success, NSError *error) {
+  PAGConfig *config = [PAGConfig shareConfig];
+  config.appID = appID;
+  config.GDPRConsent = _GDPRConsent;
+  config.doNotSell = _doNotSell;
```

Comment:
Does Pangle SDK accept `-1` for the `GDPRConsent` or `doNotSell` value?

Reply:
Yes, Pangle SDK accept -1 for GDPTConsent 、 doNotSell and childDirected.